eland/operations.py

```diff
@@ -516,7 +516,7 @@ def _unpack_metric_aggs(
                     agg_value = int(agg_value)
 
                 # If this is a non-null timestamp field convert to a pd.Timestamp()
-                elif field.is_timestamp:
+                elif field.is_timestamp and pd_agg != "count":
```

Comment:
Can change the above `elif` to be `elif pd_agg in ("nunique", "count")`

Reply:
Yep